tests/dummy/app/components/freestyle/code-block.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable ember/no-empty-glimmer-component-classes */
```

Comment:
You can delete this file.

Reply:
Thanks for the catch! https://github.com/knoxville-utilities-board/ember-nrg-ui/pull/494/commits/20e7053e7b9508edfbaf36bdb0584690d99841e7